packages/web-app-files/src/components/AppBar/Upload/FileDrop.vue

```diff
@@ -52,6 +52,7 @@ export default {
   },
   computed: {
     ...mapState(['navigation']),
+    ...mapState(['user']),
```

Comment:
Why is the user needed here?

Reply:
We now need the user in some places to build the path (`/files/username/...`). I don't know why it was needed at that exact position, but without, some action could not generate the path because no user was found.